archivers/gnutar/Portfile

```diff
@@ -50,6 +50,8 @@ post-destroot {
 
     xinstall -m 0755 -d ${destroot}${prefix}/libexec/gnubin
     ln -sf ${prefix}/bin/gnutar ${destroot}${prefix}/libexec/gnubin/tar
+    ln -sf ${prefix}/bin/gnutar ${destroot}${prefix}/bin/gtar
```

Comment:
Since your change changes the contents of the port, you will need to increment the `revision`.
```
revision            1
```